### ISSUE TITLE
Cross-File scope enhancement

### ISSUE THREAD

**Issue by Triquetra** [Gru80/obsidian-regex-replace#13]
Would it be possible to increase the scope of the search and replace to cover all files in the vault, or all files in a folder?

**Comment by DesBw** [comment]
Yes, that would be a great improvement. 
It would save me from shifting back and forth from SublimeText.

**Comment by Graywaren** [comment]
Yes, I would desperately love to be able to find/replace in the entire vault. I sometimes need to replace the file path for things such as in code blocks after I've rearranged file organization and it'd save me a lot of time to have to do it file by file each time, especially as they're in my daily notes, which makes the list of files grow every day. Doing it by folder would be a fantastic way of limiting things as well.

**Comment by Triquetra** [comment]
Perhaps a separate regex entry to define the files to be included in the scope?
